core/core-spacemacs.el

```diff
@@ -83,14 +83,9 @@ the final step of executing code in `emacs-startup-hook'.")
            (package-desc-name pkg-desc) pkg-dir))))))
 
 (defun spacemacs//lookup-load-hints (file)
-  "Findout the `load-hints' items for the file."
-  (if-let* ((load-hints)
-            ((not (member (substring file 0 1) '("/" "~")))))
-      (seq-some
-       (lambda (row)
-         (when (member file (cdr row))                 ; prefix match
-           (car row)))
-       load-hints)))
+  "Findout the `load-hints' item for the FILE."
+  (unless (member (substring file 0 1) '("/" "~"))
```

Comment:
Could maybe just use `file-name-absolute-p` here?  (Implemented in C)

Reply:
:+1:  and with `file-name-absolute-p` the function get more performance enhance. Thank you!

Reply:
applied the `file-name-absolute-p`, please help review again. Thanks